### PR TITLE
Add translations Pt. 2

### PR DESCRIPTION
**Pull request by vaxul**
Add missing translations for options.
Refine some details.

#50

### CODE REVIEW

**Comment by vaxul** [comment]
I added a first simple contribution guide

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# How to contribute
+
+**Table of content**
+* [In General](#in-general)
+* [Translations](#translations)
+
+## In General
+
+If you never contributed to an project, [this guide](https://github.com/firstcontributions/first-contributions/blob/master/README.md) is a good way to start. It's also available in multiple languages.
```
Comment (Qantas94Heavy) [inline CONTRIBUTION.md]:
```suggestion
If you've never contributed to a project before, [this guide](https://github.com/firstcontributions/first-contributions/blob/master/README.md) is a good place to start. It's also available in multiple languages.
```

File: CONTRIBUTION.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* [In General](#in-general)
+* [Translations](#translations)
+
+## In General
+
+If you never contributed to an project, [this guide](https://github.com/firstcontributions/first-contributions/blob/master/README.md) is a good way to start. It's also available in multiple languages.
+
+## Translations
+
+The translations are located in `/src/_locales/`. Beneath this directory each language is split into it's own directory. In the language directory the translations are structured in the JSON-file `messages.json`.
```
Comment (Qantas94Heavy) [inline CONTRIBUTION.md]:
```suggestion
The translations are located in `/src/_locales/`. Beneath this directory each language is split into its own directory. In the language directory the translations are structured in the `messages.json` file.
```

File: CONTRIBUTION.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+For deeper insight it's recommended to read the [Internationalization guide for web extensions](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization).
+
+### Update translation
+
+1. Open the messages.json file for the target language.
+1. Alter the `message` text for the translation to update.
+
+### Add language
+
+1. Create a directory with the short language code. Allowed are identifiers like `de_DE` or `de`. Where the latter on is the fallback.
```
Comment (Qantas94Heavy) [inline CONTRIBUTION.md]:
```suggestion
1. Create a directory with the short language code. Allowed are identifiers like `de_DE` or `de`, where the latter is the fallback.
```

**Comment by vaxul** [comment]
Thank you for your help. 🙂

**Comment by projectdelphai** [comment]
Debating moving the contributions guide into the main readme just because the readme is so short, but just merging for now. Thanks for the pull request :) I'm planning to do a release tomorrow night so it should be in the main release by then

**Comment by vaxul** [comment]
Thanks for the merge.

In my option the readme should be a short overview and should directing to the several parts. For the contribution guide I had much more in mind (as the part you removed). This is only something to start.

**Pull request merged**